### DISCUSSION TITLE
Payment is to be deducted from my withdrawal

### DISCUSSION THREAD

**Discussion by Deadliestrope** [btcpayserver/btcpayserver#5451] [Technical Support] [answered]
I request the payment due for services to be deducted from the amount I am withdrawaling?

**Comment by gutek2** [comment]
Good morning, guess me, please include the amount of your paycheck

sob., 4 lis 2023, 18:39 użytkownik Deadliestrope ***@***.***>
napisał:

**Reply by Deadliestrope** [reply]
Amount owed to me is $46904.30 us
They are requesting $64 us for the service but I have paid that 3 times and refuse to put another penny out until I receive what is owed or at least what I have lost

**Comment by Deadliestrope** [comment]
$46904.30 USD

**Comment by Kukks** [comment, marked as answer]
https://blog.btcpayserver.org/bitcoin-bonus-scam/
